torch_xla/stablehlo.py

```diff
@@ -212,6 +212,16 @@ def call_function(self, target, args: Tuple, kwargs: Dict) -> Any:
       new_kwargs['device'] = self._device
     return super().call_function(target, args, new_kwargs)
 
+  def run_node(self, n) -> Any:
```

Comment:
Could you add a test?

Reply:
I added a test, even though it requires unmerged XLA change to work.